examples/demo_actor/demo_actor/test_demo_actor.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+from demo_actor import DemoActor
+
+from dapr.actor.runtime.mock_actor import create_mock_actor
+
+
+class DemoActorTests(unittest.IsolatedAsyncioTestCase):
+    def test_create_actor(self):
+        mockactor = create_mock_actor(DemoActor, '1')
+        self.assertEqual(mockactor.id.id, '1')
+
+    async def test_get_data(self):
+        mockactor = create_mock_actor(DemoActor, '1')
+        self.assertFalse(mockactor._state_manager._mock_state)
+        val = await mockactor.get_my_data()
+        self.assertIsNone(val)
+
+    async def test_set_data(self):
+        mockactor = create_mock_actor(DemoActor, '1')
+        self.assertFalse(mockactor._state_manager._mock_state)
+        val = await mockactor.get_my_data()
+        self.assertIsNone(val)
```

Comment:
We don't need to repeat the same check as above

Reply:
This is resolved in latest commit.